csharp_dotnetcore/9.Message-Routing/Dialogs/Cancel/CancelResponses.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using MessageRoutingBot.Dialogs.Cancel.Resources;
+using Microsoft.Bot.Builder.TemplateManager;
+
+namespace MessageRoutingBot
+{
+    /// <summary>
+    /// Responses for the <see cref="CancelDialog"/>.
+    /// </summary>
+    public class CancelResponses : TemplateManager
+    {
+        // Constants
+        public const string _confirmPrompt = "Cancel.ConfirmCancelPrompt";
+        public const string _cancelConfirmed = "Cancel.CancelConfirmed";
```

Comment:
public const shouldn't have "_" and should start with a Capital Letter

Reply:
will fix!